test/TestProjectWithPrePost/RefactorLog/TestProjectWithPrePost.refactorlog

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see we've added this file, but don't we need to add it as a `<RefactorLog>` item to the `.csproj`. Then we can build the project and verify that it is embedded in the `.dacpac` as expected.

Reply:
I've added the file in the test projects file. Is this enough to test it?

Reply:
We're not currently building this project as part of CI. Perhaps you can do a `dotnet build` on the TestProjectWihPrePost and then crack open the resulting `.dacpac`  and check whether it includes the refactor.xml file.

Reply:
Verified, the dacpac file contains the `refactor.xml`
![image](https://user-images.githubusercontent.com/5229316/91959696-39e4b580-ed09-11ea-8c08-bd6678301285.png)

Reply:
Cool!